python/openvino_tokenizers/utils.py

```diff
@@ -10,15 +10,66 @@
 from openvino import Model, Type
 from openvino.preprocess import PrePostProcessor
 from openvino.runtime import opset12 as opset
+from dataclasses import dataclass, field, asdict
+from transformers import PreTrainedTokenizerBase
```

Comment:
I would avoid global import from transformers since it is an optional dependency.

Reply:
done